packages/input-segments/test/input-segments.test.ts

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+
+import '../sp-input-segments.js';
+import { InputSegments } from '..';
+import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+
+describe('InputSegments', () => {
+    testForLitDevWarnings(
+        async () =>
+            await fixture<InputSegments>(
+                html`
+                    <sp-input-segments></sp-input-segments>
+                `
+            )
+    );
+    it('loads default input-segments accessibly', async () => {
+        const el = await fixture<InputSegments>(
+            html`
+                <sp-input-segments></sp-input-segments>
+            `
+        );
+
+        await elementUpdated(el);
+
+        await expect(el).to.be.accessible();
+    });
+});
```

Comment:
We might just be testing this in implementation components, which could allow us to remove this file all together?